CHANGELOG.md

```diff
@@ -1,20 +1,26 @@
 # Changelog
 
-## UNRELEASED
+## 3.0.0-alpha.7 - 2021-04-07
 
 ### Breaks
 
 * The `trash` property has been renamed `archived`, and throughout the UI we refer to "archiving" and the "archive" rather than "move to trash" and the "trash can." A database migration is included to address this for existing databases. However, **if you set the minimumPark option, or used a boilerplate in which it is set,** you will need to **change the settings for the `parkedId: 'trash'` page to match those [currently found in the `minimumPark` option setting in the `@apostrophecms/page` source code](https://github.com/apostrophecms/apostrophe/blob/481252f9bd8f42b62648a0695105e6e9250810d3/modules/%40apostrophecms/page/index.js#L25-L32).
 
-### Fixes
+### Adds
 
+* General UX and UI improvements to the experience of moving documents to and from the archive, formerly known as the trash.
+* The `className` option can now be passed to the core widgets via page templates. If present this takes precedence over a `className` set at the module level.
 * Links to each piece are available in the manage view when appropriate.
+
+### Fixes
+
 * Bug fix: don't update the modification timestamp of a document simply because of an advisory lock, as the user might decide not to save any actual edits.
 * Prevent content loss by blocking attempts to connect Apostrophe 3.x to an Apostrophe 2.x database. Content migration tools are planned of course.
 * You can always save a draft of a new document, wherever the concept is relevant.
 * Areas nested in array schema fields can now be edited in context on the page.
 * When using `apos.image.first`, the alt attribute of the image piece is available on the returned attachment object as `._alt`. In addition, `_credit` and `_creditUrl` are available.
 * You can now set core widget `className` options in area widget configuration, not only in widget module options.
+* Do not immediately redirect to new pages and pieces.
```

Comment:
This wouldn't be a `fix` because it wasn't a bug. It would be an `add` or a `change`, yeah?